templates/app/assets/javascripts/checkout.js

```diff
@@ -18,17 +18,19 @@ window.addEventListener('DOMContentLoaded', () => {
   const termsCheckbox = document.getElementById('accept_terms_and_conditions');
 
   if (termsCheckbox) {
-    termsCheckbox.addEventListener('change', () => {
-      const submitButton = termsCheckbox.closest('form')
-        .querySelector('[type="submit"]');
-
-      if (termsCheckbox.checked) {
+    const form = termsCheckbox.closest('form')
+    const submitButton = form.querySelector('[type="submit"]');
+    form.onsubmit = function () {
+      if (!termsCheckbox.checked) {
+        termsCheckbox.scrollIntoView(); 
+        alert('Please review and accept the Terms of Service');
```

Comment:
We need to provide i18n here. I think we did something similar through `data-` attributes here:

https://github.com/solidusio/solidus_starter_frontend/blob/d3dd1e74e81cf8d6bd6dfe8cad68aa3e10c4851e/templates/app/assets/javascripts/product_selection_component.js#L17

Reply:
💯

Reply:
Forget about it, I see that everything is hardcoded to English, at this point.